docs/content/utilities/moose_docs/docs_standards/index.md

```diff
@@ -0,0 +1,59 @@
+# Standards for Documentation Pages
+
+Clear and consistent documentation is a necessary component of code developement within MOOSE.
+It is expected that developers submitting new classes to the MOOSE and MOOSE module repository will create a corresponding markdown file to document the new classes.
+Developers who modify existing classes should update the corresponding markdown documentation file.
+The standards outlined on this page are intended to aid in creating and maintaining easy-to-read, focused, and consistent documentation for the MOOSE and MOOSE module codes.
+
+## Content Creation Standards
+Developers submitting new classes to the MOOSE repository should also create a new markdown documentation files with the same names as the created classes.
+A [template documentation file](/docs_markdown_template.md) is provided as a starting place for new markdown documentation pages.
+
+### End-User Focused
+The intended audience for the markdown documentation pages is an end-user with no or little development experience.
+As such, the markdown documentation page should focus on the theoretical background and input file usage of the corresponding class, including limitations on usability ranges for the class.
+If the background and description are long, break up the information with multiple second (`##`) and third (`###`) level headings.
+Keep in mind that only the second level headings will appear in the right-hand sidebar as navigation links.
+
+### Citation Information
+When appropriate, include citation information for the models implemented in the class being documented.
+Avoid accidental duplication of the citation information in the documentation text by listing and immediately citing the source; use only `\cite{example_reference}` which will render as the author name and publication year in the documentation page.
+If a reference is not currently included in the appropriate bibtex file, add the reference citation information to the bibtex file.
```

Comment:
What bibtex file?  Did you include a bibtex file?  And i can't see where your ```example_reference``` is defined.

Reply:
I did not include a bibtex file, but I'll improve this to cite an actual bibtex file.  I'd like to keep an example reference that will throw an error if anyone, say me, forgets to remove/update the example reference in the new documentation.